config/routes.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  get 'spendings/index'
+  get 'spendings/show'
+  get 'spendings/new'
+  get 'spendings/edit'
+
+  devise_for :users
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+  root "homes#index"
+
+  resources :groups, :path => 'home'
+  resources :spendings
```

Comment:
[REQUIRED] Ensure you create only what you need in projects. Kindly limit your routes with only and except so as to create routes for only the actions that you have created. For example, in spendings you don't have a update action but resources will automatically create route for it.

Reply:
THank you for the feedback. Changes has been implemented